ledger/sandbox-classic/src/test/lib/scala/platform/sandbox/ScenarioLoadingITBase.scala

```diff
@@ -23,12 +23,13 @@ import com.daml.ledger.client.services.transactions.TransactionClient
 import com.daml.platform.sandbox.services.{SandboxFixture, TestCommands}
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.time.{Millis, Span}
-import org.scalatest.{Assertion, Suite}
+import org.scalatest.{Assertion, Ignore, Suite}
 import org.scalatest.wordspec.AsyncWordSpec
 import org.scalatest.matchers.should.Matchers
 
 import scala.concurrent.Future
 
+@Ignore
```

Comment:
Can you ignore an abstract class?

Reply:
Hmm, oversaw this. It's going to be removed anyway imminently. I'll revert anyway